base_classes/NXactuator.nxdl.xml

```diff
@@ -0,0 +1,127 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<?xml-stylesheet type="text/xsl" href="nxdlformat.xsl"?>
+<!--
+# NeXus - Neutron and X-ray Common Data Format
+#
+# Copyright (C) 2014-2024 NeXus International Advisory Committee (NIAC)
+#
+# This library is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License as published by the Free Software Foundation; either
+# version 3 of the License, or (at your option) any later version.
+#
+# This library is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this library; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
+#
+# For further information, see http://www.nexusformat.org
+-->
+<definition xmlns="http://definition.nexusformat.org/nxdl/3.1" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" category="base" type="group" name="NXactuator" extends="NXobject" xsi:schemaLocation="http://definition.nexusformat.org/nxdl/3.1 ../nxdl.xsd">
+    <doc>
+         An actuator used to control an external condition.
+         
+         The condition itself is described in :ref:`NXenvironment`.
+    </doc>
+    <field name="model">
+        <doc>
+             Actuator identification code/model number
+        </doc>
+    </field>
+    <field name="name">
+        <doc>
+             Name of the actuator
+        </doc>
+    </field>
+    <field name="short_name">
+        <doc>
+             Short name of actuator used e.g. on monitor display program
+        </doc>
+    </field>
+    <field name="attached_to">
+        <doc>
+             Describe where the actuator is attached to.
+             This could be an instance of NXsample or a device on NXinstrument.
+        </doc>
+    </field>
+    <field name="physical_quantity">
+        <doc>
+             Name for the physical quantity effected by the actuation
+             
+             Examples:
+             temperature | pH | magnetic_field | electric_field | current | conductivity | resistance | voltage |
+             pressure | flow | stress | strain | shear | surface_pressure
+        </doc>
+    </field>
+    <field name="type">
+        <doc>
+             The type of hardware used for the actuation.
+             
+             Examples (suggestions, but not restrictions):
+             
+             :Temperature: laser | gas lamp | filament | resistive
+             :Pressure: anvil cell
+             :Voltage: potentiostat
+        </doc>
+    </field>
+    <field name="OUTPUT" type="NX_FLOAT" units="NX_ANY">
```

Comment:
Add nameType="any"

Reply:
This instead: name="outputVALUE" nameType="partial"

Reply:
implemented